cms/envs/test.py

```diff
@@ -331,3 +331,7 @@
 
 # API access management -- needed for simple-history to run.
 INSTALLED_APPS += ('openedx.core.djangoapps.api_admin',)
+
+# Used with Email sending
+RETRY_ACTIVATION_EMAIL_MAX_ATTEMPTS = 5
```

Comment:
@attiyaIshaque sounds like you should import these settings from lms/envs/common.py into cms/envs/common.py too in case someone tries to reset their email from studio.